.vortex/tests/phpunit/Functional/ToolingBootstrapTest.php

```
@@ -0,0 +1,217 @@
+        $this->assertFileNotContainsString($stale_bin, 'pre-existing binary', 'The stale binary is replaced with the generated proxy.');
+      }
+    }
+
+    $this->logSubstep('Assert a repeated run short-circuits without reinstalling');
+    $canary = $project_dir . '/vendor/drevops/vortex-tooling/canary.txt';
+    File::dump($canary, "canary\n");
+    $this->cmd('bash scripts/vortex-tooling.sh', txt: 'Repeated bootstrap run must exit early');
+    $this->assertFileExists($canary, 'The repeated run does not reinstall the package.');
+
+    $this->logStepFinish();
+  }
```

PHPMD (2.15.0)

[error] The parameter $package_present is not named in camelCase. (undefined)

(CamelCaseParameterName)